lib/util.c

```diff
@@ -1460,6 +1460,61 @@ grn_inspect_limited(grn_ctx *ctx, grn_obj *buffer, grn_obj *obj)
   return buffer;
 }
 
+grn_obj *
+grn_inspect_key_with_table(grn_ctx *ctx, grn_obj *buffer, grn_obj *table, const void *key, unsigned int key_size)
+{
+  grn_obj inspected;
+
+  if (!buffer) {
+    buffer = grn_obj_open(ctx, GRN_BULK, 0, GRN_DB_TEXT);
+  }
+
+  if (!table) {
+    ERR(GRN_INVALID_ARGUMENT,
+        "failed to inspect key with table: table must not NULL");
+    return buffer;
+  }
```

Comment:
We don't need to set error.
We can just write `"(NULL)"` and return like other `grn_inspect()` family does.

Reply:
I've fixed